src/topology/algebra/ordered/basic.lean

```diff
@@ -1158,6 +1162,123 @@ mem_sets_of_superset (Ioo_mem_nhds ha hb) Ioo_subset_Ico_self
 lemma Icc_mem_nhds {a b x : α} (ha : a < x) (hb : x < b) : Icc a b ∈ 𝓝 x :=
 mem_sets_of_superset (Ioo_mem_nhds ha hb) Ioo_subset_Icc_self
 
+/-!
+### Existence of sequences tending to Inf or Sup of a given set
+-/
+
+lemma exists_seq_strict_mono_tendsto_of_is_lub_of_not_mem' {t : set α} {x : α}
```

Comment:
What about putting that lemma in the `is_lub` namespace? And similarly for the other ones of course.